libs/blocks/section-metadata/section-metadata.js

```diff
@@ -34,6 +36,39 @@ function handleTopHeight(section) {
   const headerHeight = document.querySelector('header').offsetHeight;
   section.style.top = `${headerHeight}px`;
 }
+
+export function promoIntersectObserve(el, startPoint, options = {}) {
+  let abovePromoStart = false;
```

Comment:
Since you don't need this outside the `io` scope, this can be removed from here and then L49-L51 could simply be
```
const abovePromoStart = isPromoStart
        ? (entry.isIntersecting || entry.boundingClientRect.y > 0)
        : false;
```

Reply:
@overmyheadandbody The abovePromoStart is state of the scroll that is set only when the intersection observer is triggered by promo placeholder. 
The state set by promo placeholder is used when the observer is triggered for the footer.
For case when the promobar needs to be always visible this needs to be false always. Hence we are using one variable to save the state of scroll to be used by intersection observer.
The change would make the section visible every time the page loads because this would mean that the abovePromoStart is true which would indicate to the footer that the user has scrolled past the blocks after which the section should be made visible and it would set the display block for the section on load .

Reply:
I wasn't questioning the logic, just that a variable is declared here, only to be updated on L49. But it's not a showstopper, just a minor improvement :)

Reply:
Agree with @overmyheadandbody - I don't see why `abovePromoStart` shouldn't just be declared on line 49.